app/views/styled_forecasts/_day_tab.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="tab py-4 flex-1 today mx-2 px-1 text-center daqi-low border-b-0 border-l-2 border-r-2 border-t-2 border-gray-300">
+<div class="day">
```

Comment:
So that we don't have a lot of diffs caused by indentation changes could I ask that you use the ERB Formatter extension to format your views?

https://marketplace.visualstudio.com/items?itemName=aliariff.vscode-erb-beautify

Reply:
thanks, have done